collectors/channel_collector_test.go

```diff
@@ -0,0 +1,84 @@
+package collectors
+
+import (
+	"testing"
+
+	"github.com/btcsuite/btcutil"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/stretchr/testify/require"
+)
+
+type inboundFeeTestCase struct {
```

Comment:
Do we need this struct? Couldn't we just have `testCases=[]struct{...}` then pass 3 params to `testGetInboundFee`? I don't think the indenting would be too bad.

Reply:
Not three, but five in that case. Also the `remotePolicies` and `remoteBalances`. Sure you want that?

Reply:
Did move the test cases into the function.